app/assets/scripts/components/map.js

```diff
@@ -272,23 +275,12 @@ const Map = React.createClass({
     const isQuantile = scale.hasOwnProperty('invertExtent');
     let iterable = (isQuantile ? scale.range() : scale.domain()).sort();
 
-    const nameLookup = {'very low': 5, 'low': 4, 'medium': 3, 'high': 2, 'very high': 1};
-    const idLookup = {5: 'very low', 4: 'low', 3: 'medium', 2: 'high', 1: 'very high'};
-    let convertId = true;
+    let convertId = false;
```

Comment:
IMO we're treating this as a special case, especially considering the majority of the indicators are quantile ranges, so `convertId` should be false by default.